cli/src/assembly/resources/sbin/start-cli.sh

```diff
@@ -29,7 +29,7 @@ fi
 MAIN_CLASS=org.apache.iotdb.cli.Cli
 
 
-CLASSPATH=""
+CLASSPATH="."
```

Comment:
This may prevent "class not found" when using git bash in windows to run this script.